### PR TITLE
Address UX feedback on the sidebar

### PR DESCRIPTION
**Pull request by mprahl**
Please review this commit by commit to see what is going on in the PR

### CODE REVIEW

File: src/app/story/storysidebar/storysidebar.component.ts [hunk 1]
```diff
@@ -1,21 +1,43 @@
...
 
-  constructor() { }
+  constructor(private datePipe: DatePipe) { }
 
   ngOnInit() {
     // Open the sidebar by default on larger screens
     this.sidebarOpen = window.screen.width >= 996;
   }
 
+  ngOnChanges() {
```
Comment (yashvardhannanavati) [inline src/app/story/storysidebar/storysidebar.component.ts]: Can you please add some comments here?

File: src/app/story/storysidebar/storysidebar.component.ts [hunk 2]
```diff
@@ -45,4 +43,29 @@ export class StorysidebarComponent implements OnInit, OnChanges {
     }
     return false;
   }
+
+  filterProperties(node: any): any {
```
Comment (yashvardhannanavati) [inline src/app/story/storysidebar/storysidebar.component.ts]: Just out of curiosity, can you please tell me why the Pipe was converted into a method?

**Comment by yashvardhannanavati** [comment]
Just a single comment to add comments :p Other than that :+1:

File: src/app/story/storysidebar/storysidebar.component.ts (hunk 1 above)
Comment (mprahl) [reply src/app/story/storysidebar/storysidebar.component.ts]: Added

**Pull request merged**